### PR TITLE
Added metadata info for Disk Space widget

### PR DESCRIPTION
**Pull request by marmarta**
requires https://github.com/QubesOS/qubes-core-admin/pull/278
fixes QubesOS/qubes-issues#5053

### CODE REVIEW

File: qui/tray/disk_space.py [hunk 1]
```diff
@@ -33,6 +33,14 @@ def __populate_pools(self):
                 self.warning_message.append(
                     "\n{:.1%} space left in pool {}".format(
                         1-pool.usage/pool.size, pool.name))
+            if 'metadata_size' in pool.usage_details:
+                metadata_usage = pool.usage_details['metadata_usage'] / \
+                                 pool.usage_details['metadata_size']
```
Comment (marmarek) [inline qui/tray/disk_space.py]: 1. Check for `metadata_size` being 0.
2. You won't like it, but you might also need to add `usage_details` property to `Pool` class in qubesadmin module (client side of Admin API). But the problem is at Admin API there is one call for all the info. And it lands in pool.config dict. I don't see any easy and reliable (independent of storage driver) way to extract "usage" info from other info. Some heuristic would be looking for "usage" and "size" substrings, but that's ugly :/ Maybe force some prefix in admin api call output? Or rename `config` to `info` (keeping compatibility wrapper) and don't distinguish them at all?

Comment (marmarta) [reply qui/tray/disk_space.py]: what do you mean by 'might'?

Comment (marmarek) [reply qui/tray/disk_space.py]: There are alternatives, listed in this comment.

File: qui/tray/disk_space.py [hunk 2]
```diff
@@ -33,6 +33,15 @@ def __populate_pools(self):
                 self.warning_message.append(
                     "\n{:.1%} space left in pool {}".format(
                         1-pool.usage/pool.size, pool.name))
+            if 'metadata_size' in pool.usage_details and\
+                    pool.usage_details['metadata_size']:
```
Comment (marmarek) [inline qui/tray/disk_space.py]: I think `if pool.usage_details.get('metadata_size', None)` would be nicer.

File: qui/tray/disk_space.py (hunk 1 above)
Comment (marmarta) [reply qui/tray/disk_space.py]: tbh I'm not sure what do you mean by the words there. is it documented anywhere in more detail?

Comment (marmarek) [reply qui/tray/disk_space.py]: Admin API documentation and basic concepts: https://www.qubes-os.org/doc/admin-api/
Server side (qubesd service) lives in core-admin repo and you added relevant part here: https://github.com/QubesOS/qubes-core-admin/pull/278/files#diff-0c889398bb7fbe645fcda185af7132de
Client side (qubesadmin python module) lives in core-admin-client, documentation here: https://dev.qubes-os.org/projects/core-admin-client/en/latest/qubesadmin.html#qubesadmin.storage.Pool

In manager you have objects from qubesadmin module, client side of Admin API. Changes in server side Admin API in most cases require also adding relevant part to its client side.

**Pull request merged**